jobs/api/v4/JobsSample/CreateClientEventSample.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using static Google.Cloud.Talent.V4Beta1.JobEvent.Types;
+
+using Google.Cloud.Talent.V4Beta1;
+using Google.Protobuf.WellKnownTypes;
+using System;
+
+namespace GoogleCloudSamples
+{
+    internal class CreateClientEventSample
+    {
+        // [START job_search_create_client_event]
+        public static object CreateClientEvent(string projectId, string tenantId, string requestId, string eventId)
```

Comment:
```suggestion
        public static object CreateClientEvent(string projectId, string tenantId, string requestId, string eventId, params string[] jobIds)
```

And then you can call this method as follows

```
        CreateClientEvent("projectId", "tenantId", "requestId", "eventId", "job1", "job2", "job3");
```